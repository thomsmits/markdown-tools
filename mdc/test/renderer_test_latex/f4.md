# Chapter
## Slide

  * Link: [LINK_WITHUNDERSCORE](https://www.example.com)
