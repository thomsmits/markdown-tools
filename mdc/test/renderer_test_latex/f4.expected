# Chapter
## Slide

\begin{ul1}
\item Link: \href{https://www.example.com}{LINK\_WITHUNDERSCORE}
\end{ul1}

